js/components/date.js

```diff
@@ -23,6 +23,11 @@ Fliplet.FormBuilder.field('date', {
       default: true
     }
   },
+  data() {
```

Comment:
```suggestion
  data: function() {
```

Reply:
@squallstar @upplabs-alex-levchenko What tests or linting do we have to prevent this being merged ever again?